### ISSUE TITLE
The error occurred in the Stop (Missing request parameters) step.

### ISSUE THREAD

**Issue by kiteboy** [openhomeautomation/yun-weather-temboo#3]
I have the sensors test woorking fine but the temboo log just doesnt work for me

I get the following error when debug is on

Calling the AppendRow Choreo...
Error
A Step Error has occurred: "An input error has occurred. RowData and SpreadsheetTitle are required.".  The error occurred in the Stop (Missing request parameters) step

Contacted temboo and they said the following:

It seems like you could be running into an out-of-memory condition on your board (a common issue on resource constrained devices). This can cause the Choreo data to get overwritten and corrupted, which is why it appears that there is a problem with your Temboo code even though it looks correct. 
You can reduce memory usage by using a Profile to store your API credentials on Temboo. This means that you can simply reference them from your sketch without including all of the String data that consumes memory. You can learn more about that approach here: 
https://temboo.com/hardware/profiles
You can also reduce RAM by removing any unnecessary print statements in your sketch.

Is the code too big??

**Comment by marcoschwartz** [comment]
Indeed, it's a quite big sketch and it just fits into the Arduino, which sometimes leads to memory issues. Indeed I'll try either deactivating some part of the sketch or removing Serial.print() statements.
